cross/gogs/PLIST

```diff
@@ -0,0 +1,1077 @@
+gogs/LICENSE
```

Comment:
use prefixes (see https://github.com/SynoCommunity/spksrc/wiki/PLIST-files) and delete stuff not needed (Readme, licence etc)